app-ktor-server/src/main/kotlin/com/ustadmobile/lib/rest/StartFile.kt

```diff
@@ -0,0 +1,53 @@
+package com.ustadmobile.lib.rest
+
+import com.ustadmobile.core.catalog.contenttype.EpubTypePluginCommonJvm
+import com.ustadmobile.core.contentjob.ProcessContext
+import com.ustadmobile.core.db.UmAppDatabase
+import com.ustadmobile.core.util.DiTag
+import com.ustadmobile.door.DoorUri
+import com.ustadmobile.door.ext.DoorTag
+import com.ustadmobile.lib.db.entities.ContentJob
+import com.ustadmobile.lib.db.entities.ContentJobItem
+import com.ustadmobile.lib.db.entities.ContentJobItemAndContentJob
+import io.ktor.application.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import org.kodein.di.DI
+import org.kodein.di.direct
+import org.kodein.di.instance
+import org.kodein.di.ktor.closestDI
+import org.kodein.di.on
+import java.io.File
+import java.nio.file.Files
+
+fun Route.StartFile(){
+
+    get("/startFile"){
+
+        val di : DI by closestDI()
+        val repo: UmAppDatabase = di.on(call).direct.instance(DoorTag.TAG_REPO)
+        val epubPlugin: EpubTypePluginCommonJvm = di.on(call).direct.instance()
+
+        val epub = File("/home/ustad/server/test.epub")
```

Comment:
What is this doing?! This looks like testing code of some kind... Shouldn't this be deleted?